File: Python/program/Human.py
# class declaration
class Human:

    # atribut private
    __nik = ""
    __nama = ""
    __jenis_kelamin = ""

   # konstruktor
    def __init__(self, nik="", nama="", jenis_kelamin=""):
        self.__nik = nik
        self.__nama = nama
        self.__jenis_kelamin = jenis_kelamin

    # Getter and Setter

    def get_nik(self):
        return self.__nik

    def set_nik(self, nik):
        self.__nik = nik

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_jenis_kelamin(self):
        return self.__jenis_kelamin

    def set_jenis_kelamin(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin

File: PHP/program/Human.php
<?php

// deklarasi kelas Human
class Human {
    // atribut private
    private $nik = '';
    private $nama = '';
    private $jenis_kelamin = '';

    // konstruktor
    public function __construct( $nik = '', $nama = '' , $jenis_kelamin = '') {
        $this->nik = $nik;
        $this->nama = $nama;
        $this->jenis_kelamin = $jenis_kelamin;
    }
 /* Getter dan Setter */
    public function set_nik($nik){
        $this->nik = $nik;
    }

    public function get_nik(){
        return $this->nik;
    }
    
    public function set_nama($nama){
        $this->nama = $nama;
    }
    public function get_nama(){
        return $this->nama;
    }
    public function set_jenis_kelamin($jenis_kelamin){
        $this->jenis_kelamin = $jenis_kelamin;
    }

    public function get_jenis_kelamin(){
        return $this->jenis_kelamin;
    }

}

?>